wrappers/python/simtk/openmm/app/gromacstopfile.py

```diff
@@ -857,19 +880,44 @@ def createSystem(self, nonbondedMethod=ff.NoCutoff, nonbondedCutoff=1.0*unit.nan
                         continue # We'll use the automatically generated parameters
                     atom1params = nb.getParticleParameters(baseAtomIndex+atoms[0])
                     atom2params = nb.getParticleParameters(baseAtomIndex+atoms[1])
-                    exceptions.append((baseAtomIndex+atoms[0], baseAtomIndex+atoms[1], atom1params[0]*atom2params[0]*fudgeQQ, params[0], params[1]))
+                    # enable tracking pairs and exceptions separately
+                    if self._defaults[1] == '1':
+                        pairs.append((baseAtomIndex+atoms[0], baseAtomIndex+atoms[1], atom1params[0]*atom2params[0]*fudgeQQ, params[0], params[1]))
+                    elif self._defaults[1] == '2':
+                        exceptions.append((baseAtomIndex+atoms[0], baseAtomIndex+atoms[1], atom1params[0]*atom2params[0]*fudgeQQ, params[0], params[1]))
```

Comment:
I'm a bit confused by the above lines.  In mode 1, don't you still need to add nonbonded exclusions for these (in addition to adding terms to the CustomBondForce)?